Kernel/Prekernel/Arch/aarch64/init.cpp

```diff
@@ -24,7 +25,16 @@ extern "C" [[noreturn]] void init()
     uart.print_num(firmware_version);
     uart.print_str("\r\n");
 
-    halt();
+    auto& timer = Prekernel::Timer::the();
+    u64 start_musec = 0, now_musec;
```

Comment:
This style is rarely used in the codebase (probably for readability reasons), let's declare those one at a time:

```suggestion
    u64 start_musec = 0;
    u64 now_musec;
```